### DISCUSSION TITLE
Git LFS, does Github repo size include them?

### DISCUSSION THREAD

**Discussion by bitinn** [community/community#22926] [New to GitHub]
<p>Hi all,</p>
<p>I really need to find out which of my repo is using most Git LFS storage, it seems this page:</p>
<p><a href="https://github.com/settings/repositories" rel="nofollow noopener">https://github.com/settings/repositories</a></p>
<p>Doesn’t count LFS usage.</p>
<p>If true, how can I figure out which of my repo is using too much LFS storage?</p>
<p>Thx,<br>
David</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi David,</p>
<p>The best way to find out which repo is using your LFS is to <a href="https://github.com/contact" rel="nofollow noopener">write to private support</a>. They’ll be able to look at your account specifics and help you.</p>
<p>Cheers!</p>
